Configuration/TCA/tx_bootstrappackage_accordion_item.php

```diff
@@ -28,7 +28,6 @@
         'hideTable' => true,
         'hideAtCopy' => true,
         'prependAtCopy' => 'LLL:' . $generalLanguageFile . ':LGL.prependAtCopy',
-        'transOrigPointerField' => 'l10n_parent',
```

Comment:
Why do you remove this definition? Can you show me the related core change please or give me another good reason.